resources/twitchcommands.py

```diff
@@ -192,13 +192,17 @@ def max_val(val, min, max):
               "swamp-cave2","swamp-game","swamp-cave3","ogre-start","ogre-race","ogre-end",
               "village3-start","village3-warp","village3-farside","maincave-start",
               "maincave-to-darkcave","maincave-to-robocave","darkcave-start","robocave-start",
-              "robocave-bottom","snow-start","snow-fort","snow-snow-flut-flut","snow-pass-to-fort",
+              "robocave-bottom","snow-start","snow-fort","snow-flut-flut","snow-pass-to-fort",
               "snow-by-ice-lake","snow-by-ice-lake-alt","snow-outside-fort","snow-outside-cave",
               "snow-across-from-flut","lavatube-start","lavatube-middle","lavatube-after-ribbon",
               "lavatube-end","citadel-start","citadel-entrance","citadel-warp","citadel-launch-start",
               "citadel-launch-end","citadel-generator-start","citadel-generator-end","citadel-plat-start",
               "citadel-plat-end","citadel-elevator","finalboss-start","finalboss-fight"]
 
+lang_list = ["english","french","german","spanish","italian","japanese","uk-english"]
+input_list = ["square","circle","x","triangle","up","down","left","right"]
+cam_list = ["endlessfall","eye","standoff","bike","stick"]
```

Comment:
Do we need `string` to replace `normalcam`? Or I guess the idea is that the player has to wait out the duration to get back to normal cam?

Reply:
normalcam was used to put it back to normal after invertcam. I removed it because it became kind of useless since it fixes itself after the duration ends. And yeah it goes back to string after the cam command ends, but adding string in the list isn't a bad idea if you want to help the player out and you have the cooldown set to lower than the duration for some reason